recipes/addons.rb

```diff
@@ -102,3 +102,42 @@
   include_recipe "kube-hops::kserve"
 end
 include_recipe "kube-hops::hopsmon"
+
+directory "#{node['kube-hops']['fuse']['assets_dir']}" do
+  owner node['kube-hops']['user']
+  group node['kube-hops']['group']
+  mode '0700'
+  action :create
+end
+
+
+
+# create and apply yml files required for fuse related stuff
+smart_device_manager_file = "smart-device-manager-plugin.yml"
+hopsfsmount_apparmour_profile = "hopsfsmount-apparmour-profile.yml"
+
+template "#{node['kube-hops']['fuse']['assets_dir']}/#{smart_device_manager_file}" do
+  source "smart-device-manager-plugin.yml.erb"
+  owner node['kube-hops']['user']
+  group node['kube-hops']['group']
+end
+
+template "#{node['kube-hops']['fuse']['assets_dir']}/#{hopsfsmount_apparmour_profile }" do
```

Comment:
I'm not picky for typos in comments but in this case it is important because if I want to search in the codebase where the profile is used and I search for `apparmor` I will not find anything.